.github/workflows/ci-backend.yml

```diff
@@ -267,18 +267,18 @@ jobs:
         steps:
             - name: Fetch posthog-cloud
               run: |
-                  curl -L https://github.com/posthog/posthog-cloud/tarball/master | tar --strip-components=1 -xz --
+                  curl -L https://github.com/posthog/posthog-cloud/tarball/settings-refactor | tar --strip-components=1 -xz --
```

Comment:
Reminder to revert this back to master

Reply:
100% - also in the task list to do before merging :) Just wanted to show the tests would be green